### DISCUSSION TITLE
Upayments

### DISCUSSION THREAD

**Discussion by Husseindhaher** [woocommerce/woocommerce#48419] [Checkout Flow]
an error shown to me regarding the Upayments payment gate way , that it is not compatible with the checkout page blocks , is there an solution .

**Comment by nielslange** [comment]
Hello @Husseindhaher 👋

The Checkout blocks require a different payment gateway integration than the shortcode-based checkout. You can find information on how to make your payment gateway compatible with the Checkout block here:
- https://sevengits.com/payments-with-woocommerce-checkout-blocks/
- https://medium.com/@hizanaoa40463/woocommerce-blocked-checkout-issue-solution-63341e043c29
- https://rudrastyh.com/woocommerce/checkout-block-payment-method-integration.html
